settings/ticket.rst

```diff
@@ -149,3 +149,24 @@ the desired agents in the ``Exception Users`` list.
 
       The search function in this area is only effective, if you have too many
       agents to display at once to help you with the configuration.
+
+Notifications
+-------------
+
+Default Notifications
+   This setting allows you to configure the default ticket notifications that
+   will be applied to all new agent users. Simply change the notification matrix
+   to desired state and press **Save**.
```

Comment:
Does `new agent users` refer to both new agent users and old users who just got assigned agent permission? Maybe we should explain that this applies in both cases?